src/domain/resource/utils.js

```diff
@@ -436,9 +409,10 @@ const isBetweenReservedTimeRange = (events, start, end) => {
  * @param start {Date|string} Either a Date object or date string that can be parsed as moment object.
  * @param end {Date|string} Either a Date object or date string that can be parsed as moment object.
  * @param isStaff {boolean} Staff users have permission to bypass maxPeriod check.
+ * @param events {Array} Reserved reservations from resource data.
  * @returns {boolean}
  */
-export const isTimeRangeReservable = (resource, start, end, isStaff = false, events) => {
+export const isTimeRangeReservable = (resource, start, end, events) => {
```

Comment:
Was the argument `isStaff` removed by purpose? There is still code using that argument.

Reply:
Yes, this function no longer need to care about `min_period` or `max_period` anymore
Passing it is useless.

Staff member still have to respect existed reservation (no overlapping)

Reply:
Ah sheet, forgot to remove, nice find! 👍